configure.sh

```diff
@@ -9,17 +9,17 @@
 #   script depends on the arguments passed:
 #   
 #   1. If called without arguments, CMake scripts for a release (optimized) build
-#      of ChiTech will be generated in the directory chi_build. Additional
+#      of ChiTech will be generated in the directory `build`. Additional
```

Comment:
ChiTech mention here

Reply:
I did not change references to ChiTech in the code, yet. Just in the build script. For code, a lot more work will be required with file renames, etc.

Reply:
I guess, you see `configure.sh` as part of the build infrastructure, I can add a patch to fix it in this file. For the code changes, I would personally go with a separate PR. IMO PRs should be as small as possible and do just 1 thing.

Reply:
One more thing, what is final agreement of the capitalization of the code name? IIRC, there were 3 variants:
- OpenSn
- openSn
- opensn

If there is no agreement, yet. No point in changing that in this PR :-)

Reply:
Yes, I was thinking the .sh can be done now as I've seen you also added opensn so the configure.sh should remove the old name.

Reply:
A non unanimous majority likes: openSn for the code's name so we're going with that.

Reply:
Ok. I will add a patch to change the code name in `configure.sh`.